plutus-metatheory/src/Untyped/CEK.lagda.md

```diff
@@ -485,8 +485,8 @@ step (s ; ρ ▻ ƛ t)       = s ◅ V-ƛ ρ t
 step (s ; ρ ▻ (t · u))   = (s , -· ρ u) ; ρ ▻ t
 step (s ; ρ ▻ force t)   = (s , force-) ; ρ ▻ t
 step (s ; ρ ▻ delay t)   = s ◅ V-delay ρ t
-step (s ; ρ ▻ con (tmCon t c)) = s ◅ V-con t c
-step (s ; ρ ▻ builtin b) = s ◅ ival b
+step (s ; ρ ▻ con (tmCon t c)) = s ◅ V-con (con t c
+step (s ; ρ ▻ builtin b)) = s ◅ ival b
```

Comment:
@michaelpj I stumbled over this during the merge. I'm not sure I follow the logic behind the left open paren on L488, and the closed on on L489.

I believe this is now gone _after_ my merge. But I'm unsure if I did that right.

Reply:
@kwxm this one.

Reply:
That's definitely wrong.  I'm not sure how it got like that.

Reply:
I saw this when I reviewed it, but when I checked out the file it was fine. Might be a glitch in the diff generation?